CONTRIBUTORS.md

```diff
@@ -0,0 +1,77 @@
+<html>

```

Comment:
@Pranav-Rastogi I have been a bit busy these days, I will see the PR next weekend and I will add some commits if necessary. So you can consider this PR closed and you can see for other issues or features to contribute to.
Cheers man 🍻 

/cc @mbad0la